sdk/core/core-http/test/xmlTests.ts

```diff
@@ -9,20 +9,22 @@ describe("XML serializer", function() {
   describe("parseXML(string)", function() {
     it("with undefined", async function() {
       const error: Error = await msAssert.throwsAsync(parseXML(undefined as any));
-      assert.notStrictEqual(
-        error.message.indexOf("Document is empty"),
-        -1,
-        `error.message ("${error.message}") should have contained "Document is empty"`
+      assert.ok(
```

Comment:
is it worth checking this message? Should we just assert that it throws?

Reply:
I wanted to keep the old behavior of verifying that it's a parsing error.